include/mbedtls/pk.h

```diff
@@ -199,7 +287,13 @@ void mbedtls_pk_free( mbedtls_pk_context *ctx );
  *                  MBEDTLS_ERR_PK_BAD_INPUT_DATA on invalid input,
  *                  MBEDTLS_ERR_PK_ALLOC_FAILED on allocation failure.
  *
- * \note            For contexts holding an RSA-alt key, use
+ * \note            Engines that implement of opaque keys may offer an
```

Comment:
Somehow the old note is marked as outdated, so here again: `of` should be removed.

Reply:
Done